locales/en-backend.json5

```
@@ -298,6 +339,27 @@
       female: 'Female',
       male: 'Male',
       nonbinary: 'Non-Binary',
+
+      Group1: '0 - 4 Years',
+      Group2: '5 - 14 Years',
+      Group3: '15 - 34 Years',
+      Group4: '35 - 59 Years',
+      Group5: '60 - 79 Years',
+      Group6: 'Above 80 Years',
+      Total: 'Total',
     },
   },
+  interventions: {
+    'School closure': 'School closure',
+    'Face masks & social distancing School': 'Face masks & social distancing School',
+    'Face masks & social distancing Work': 'Face masks & social distancing Work',
+    'Face masks & social distancing Other': 'Face masks & social distancing Other',
+    'Remote work': 'Implementation of remote work policies',
```

Same as `de-backend.json5`.
Just "Remote work policies" to match the "School closure(s)"? 
```suggestion
    'Remote work': 'Remote work policies',
```